src/elements/content-sidebar/activity-feed/annotation-thread/useRepliesAPI.js

```diff
@@ -65,7 +66,7 @@ const useRepliesAPI = ({
         const replyData = {
             id: uuid,
             tagged_message: message,
-            type: 'comment',
+            type: FEED_ITEM_TYPE_COMMENT,
```

Comment:
I think we plan to extract this component from the feed directory at some point, right? Should we keep them separate?